woocommerce/woocommerce-android
# [Android Wear] Fix Order Details failing to handle Orders with no products

Summary
==========
Fix issue #12672 by adjusting the correct support of the Order Details screen to Orders with no products. The current implementation was expecting every Order to contain at least one product. This PR fixes it by supporting empty lists of Product as a valid state.

How to Test
==========
1. Create a Order with no products. E.g. w/ Custom amounts only
2. Open the Order in the Wear app
3. Verify that a `No products found` message is displayed instead of a error instruction.

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
